src/tests/meta-assistant/vacationCalculationTest.test.ts

```diff
@@ -0,0 +1,30 @@
+import { DateTime } from "luxon";
+import TimeUtilities from "src/meta-assistant/generic/time-utils";
+import {
+  expectedResults,
+  forecastMockVacationTypes,
+  mockNonProjectTimes
+} from "./__mocks__/forecastMocks";
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe("Expected time left test, including different non project time types", () => {
+  const fakeStartDate = DateTime.fromISO("2024-03-18");
+  const fakeEndDate = DateTime.fromISO("2024-03-21");
+
+  describe.each(expectedResults)("Check vacation calculations", (fakePersonId, expectedResult) => {
```

Comment:
Do I understand correctly this is checking that the function returns the amount of non_project_time/ time off?

Reply:
Yes, you are correct. However, this function not only checks the precise amount, but also checks if the calculations are correct.